layouts/partials/article.html

```diff
@@ -5,7 +5,8 @@
 				</header>
 {{ end }}
 {{ if or (eq $.mode "full") (eq $.mode "body") }}
-				{{ .TableOfContents }}
+				{{ $globalShowToc := default true (index .Site.Params "toc") }}
+				{{ if default $globalShowToc (index .Params "toc") }}{{ .TableOfContents }}{{ end }}
```

Comment:
Buy the way, I think using the function `isset` is more appropriate than using `index` in this case.